src/actor/mod.rs

```diff
@@ -36,6 +36,29 @@ fn actor_claims(wasm: impl AsRef<[u8]>) -> Result<jwt::Claims<jwt::Actor>> {
     Ok(claims.claims)
 }
 
+/// Returns true if the host call is authorized and false otherwise
+#[instrument]
+fn authorized_call(
```

Comment:
nit: this seems to return a `Result<()>` so it might be more appropriate to call it `authorize_call` and note that it returns `Ok(())` if authorized, and an error otherwise

Reply:
yeah was just about to suggest that this be a verb not a noun, but that's a super-nit

Reply:
Yeah, I changed it from `bool`, good point